components/dialogs/AuthDialog.vue

```diff
@@ -343,7 +343,7 @@ export default {
       errorCode: '',
       error: undefined,
       isSigningInWithEmail: false,
-      isSignIn: this.$route.query.login === '1',
+      isSignIn: this.$route.name === 'in-login',
```

Comment:
isSignIn should be default, so condition should be `$route.name === 'in-register' || query.register === '1'`